Sources/Load.php

```diff
@@ -504,7 +504,7 @@ function loadUserSettings()
 		if (empty($cache_enable) || $cache_enable < 2 || ($user_settings = cache_get_data('user_settings-' . $id_member, 60)) == null)
 		{
 			$request = $smcFunc['db_query']('', '
-				SELECT mem.*, COALESCE(a.id_attach, 0) AS id_attach, a.filename, a.attachment_type
+				SELECT mem.*, COALESCE(a.id_attach, 0) AS id_attach, a.filename, a.attachment_type, a.width "attachment_width", a.height "attachment_height" 
```

Comment:
Please us "AS" to follow the SQL structure.

Reply:
as shoud be like outer only used when needed and like outer it never needed.
but to make you happy i added this.

Reply:
By using "AS" I mean following the same SQL structure the query is already using:

`
, a.width AS attachment_width, a.height  AS attachment_height
`